app/src/androidTest/java/com/duckduckgo/app/tabs/model/TabDataRepositoryTest.kt

```diff
@@ -430,6 +428,15 @@ class TabDataRepositoryTest {
             .build()
     }
 
+    private fun mockDatabase(): TabsDao {
+        whenever(mockDao.flowDeletableTabs())
+            .thenReturn(daoDeletableTabs.consumeAsFlow())
+        whenever(mockDao.liveTabs())
```

Comment:
Test were failing because liveTabs() was null. This fixed the issue, but I did a quick clean up to avoid having some methods using logic inside `@before` and others creating their own setup.